### DISCUSSION TITLE
how to use this in ...

### DISCUSSION THREAD

**Discussion by naxsisflyby** [SWivid/F5-TTS#123] [Q&A]
Hi.

i have some servers but doesn't have gpu in it also i got labtop but doesn't have nvidia gpu ( poor intel gpu  ) in it.

How can i use this for cpu only. What kinda changes i should have to make, what is the installing pipeline. 

Please someone help me about that.

Thanks.

**Comment by NehaB18** [comment]
I can use the repo for inference and training on Macbook without any issue. Just had to set the flag PYTORCH_ENABLE_MPS_FALLBACK=1 for missing torch operator implementation for MSP backend.

**Comment by KaiKenju** [comment]
you can train by GG Colab

**Reply by naxsisflyby** [reply]
İ hear for the first time but i try to thx for information.
